.github/workflows/swift.yml

```
@@ -1,19 +1,162 @@
+    - name: Test UIKit Google Maps project
+      working-directory: ./UberCarAnimation
+      run: |
+        xcodebuild test \
+          -workspace UberCarAnimation.xcworkspace \
+          -scheme UberCarAnimation \
+          -destination 'platform=iOS Simulator,name=iPhone 15,OS=latest' \
+          -enableCodeCoverage YES \
+          CODE_SIGN_IDENTITY="" \
+          CODE_SIGNING_REQUIRED=NO \
+          ONLY_ACTIVE_ARCH=NO
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)